### DISCUSSION TITLE
Kubernetes audit logs

### DISCUSSION THREAD

**Discussion by therus000** [vectordotdev/vector#15762] [Q&A]
is there way to use a logs from audit-logs as a source in vector
and what the best type to use to sink to kafka

**Comment by jszwedko** [comment]
Hi @therus000 !

Not yet. This is being tracked by https://github.com/vectordotdev/vector/issues/1424

**Reply by cassanellicarlo** [reply]
What's the status here? How can be audit logs for kubernetes / openshift configured as source in vector?

Is file source a good option?
